src/platform/packages/shared/kbn-esql-ast/src/commands_registry/options/recommended_queries/index.ts

```diff
@@ -28,7 +37,7 @@ export const getRecommendedQueriesTemplates = ({
   timeField?: string;
   categorizationField?: string;
 }) => {
-  const queries = [
+  const queries: QueryTemplate[] = [
```

Comment:
These will always be SuggestionCategory.RECOMMENDED_QUERY_SEARCH, why don't you put this in the helpers who give the suggestions?

getRecommendedQueriesTemplatesFromExtensions and getRecommendedQueriesSuggestionsFromStaticTemplates

Reply:
Sorry I think I am not following you. I didnt suggest that, this is def wrong 🤔 

My suggestion was instead of putting this there to put it here

getRecommendedQueriesTemplatesFromExtensions and getRecommendedQueriesSuggestionsFromStaticTemplates